### DISCUSSION TITLE
android api level change or old devices support

### DISCUSSION THREAD

**Discussion by Youcefgg22** [skyline-emu/skyline#2208] [Ideas]
I wish this emulator were compatible with older devices ( android 7 or above) because some old android devices have very good specs ( I have a Samsung galaxy S7 snapdragon), So can you make this emulator compatible with older devices ?

**Comment by TheASVigilante** [comment]
It's currently impossible due to reliance on features that are exclusive to newer android versions

**Comment by ghost** [comment]
I'm pretty sure that Skyline only supports Android 10 and up due to certain requirements
